### PR TITLE
next release

### PR DESCRIPTION
**Pull request by abbaseya**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,3 +29,9 @@
 1. Start demo server: `yarn demo:nodejs:start`
 2. Build packages: `yarn build`
 3. Test demo app: http://localhost:3003
+
+### NestJS
```
Comment (clllaur) [inline README.md:33]: NextJs :)

File: packages/api/src/api-manager.ts [hunk 2]
```diff
@@ -220,15 +226,16 @@ export class ApiManager implements ApiManagerInterface {
           '[project_id]',
           this._projectId.toString()
         ),
-        route: `/track/${this._accountId}/${this._projectId}`
+        route: `/track/${this._sdkKey}`
```
Comment (clllaur) [inline packages/api/src/api-manager.ts:229]: was the metrics endpoint updated to support this?(I forgot)

**Review by clllaur — APPROVED** [review]
I see this also contains locations triggers, hopefully you tested this well to not break anything..

File: README.md (hunk 1 above)
Comment (abbaseya) [reply README.md:33]: Actually that's NestJS yes, a demo for a customer at https://app.asana.com/0/1188284429419466/1207509256438893/f

File: packages/api/src/api-manager.ts (hunk 2 above)
Comment (abbaseya) [reply packages/api/src/api-manager.ts:229]: No idea!

**Comment by abbaseya** [comment]
> I see this also contains locations triggers, hopefully you tested this well to not break anything..

Yes, it does not conflict with current logic.

File: packages/api/src/api-manager.ts (hunk 2 above)
Comment (clllaur) [reply packages/api/src/api-manager.ts:229]: oh, I wasnt even aware that nestJs exist, I only new about nextJs :))

Comment (abbaseya) [reply packages/api/src/api-manager.ts:229]: We're all good, I had the SDK looking for the `sdkKey` in config, if not found, fallback to `accountId/projectId`:
```typescript
this._sdkKey = config?.sdkKey || `${this._accountId}/${this._projectId}`;
```

**Pull request merged**